config/locales/details.pdf.yml

```diff
@@ -0,0 +1,56 @@
+en:
```

Comment:
I've pushed a refactor to improve this considerably. Now only those translations that need to be different are defined, and the ones identical will be used from the canonical details.yml